.travis.yml

```diff
@@ -8,29 +8,22 @@ dist: trusty
 addons:
   firefox: 46.0.1
 
-language: python
-python:
-  - '2.7'
+# Setting language=generic to prevent Travis from setting up a virtualenv.
+# Using a virtualenv conflicts with the global pip installation currently
+# used for configuring SecureDrop, both in development and staging.
+language: generic
 
-virtualenv:
-  system_site_packages: true
 before_install:
   - 'for e in /.packer-env/*; do echo -n "${e}: "; cat "${e}"; done'
+  # Removes Travis-specific PATH customizations that affect Python.
+  - export PATH="$(echo $PATH | tr ':' "\n" | sed '/\/opt\/python/d' | tr "\n" ":" | sed "s|::|:|g")"
   - printenv | sort # dump Travis environment for debugging
-  - sudo apt-get update -qq
-  - sudo apt-get install --yes rng-tools
-  - sudo rm -f /dev/random
-  - sudo mknod -m 0666 /dev/random c 1 9
-  - echo HRNGDEVICE=/dev/urandom | sudo tee /etc/default/rng-tools
-  - sudo /etc/init.d/rng-tools restart
-install:
-  - pip freeze -l
-  # Preinstalled travis pytest is not tracked by us
-  # Preinstalled travis pbr is not tracked by us
-  - pip uninstall pytest pbr -y
-  - pip install -r testinfra/requirements.txt
   - pip freeze -l
-script:
+install:
+  # Installing Python dependencies globally, to match SecureDrop deployment.
+  - sudo -H pip install -r securedrop/requirements/develop-requirements.txt
+  # Run linting early, to fail fast.
+  - make --keep-going lint
```

Comment:
TIL about `make --keep-going`!

Reply:
Credit goes to @dachary in #2233, I just converted it the long-form flag so folks don't have to `man` it.